### ISSUE TITLE
The GANterbury tales

### ISSUE THREAD

**Issue by brondle** [NaNoGenMo/2019#79]
I'm working on a collection of short stories built around a GPT-2 model finetuned on the original Middle English text of the Canterbury Tales. Some initial code and cleaned text are [here](https://github.com/brondle/ganterbury-tales), plus a cleaning script I used to get rid of a bunch of footnotes if that's useful for anyone. Working on an initial retraining now, I'll check back in here soon!

**Comment by brondle** [comment]
I'm finished! A few different versions are in the [repo](https://github.com/brondle/ganterbury-tales/tree/master/texts) - I initially trained for 20,000 steps and found that it was heavily overfitted. Tried 10K and 15K versions and it worked a bit better. I used a method of pulling a sentence from the original text as a seed, generating a few paragraphs from GPT-2, getting a character count from the generated text, moving to that point in the original, and using the nearest sentence as a seed.
